packages/messaging-api-messenger/src/Messenger.js

```diff
@@ -90,7 +91,7 @@ function createMessageFormData(
   const formdata = new FormData();
 
   formdata.append('message', JSON.stringify(message));
-  formdata.append('filedata', filedata);
+  formdata.append('filedata', filedata, omit(options, ['quick_replies']));
```

Comment:
omit `quick_replies` or pick `filename`?

Reply:
It looks like we should allow all of possible fields:

```js
form.append('file', stdout, {
    filename: 'unicycle.jpg', // ... or:
    filepath: 'photos/toys/unicycle.jpg',
    contentType: 'image/jpeg',
    knownLength: 19806
  });
```

So just pass through other unknown keys.

Reply:
ok lgtm